gulpTasks/test.js

```diff
@@ -41,6 +42,10 @@ gulp.task('buildTest', shell.task([
   'node node_modules/webpack/bin/webpack.js --config ./webpackConfigFiles/webpack.test.config.js'
 ]));
 
+gulp.task('uploadCoverage', ['lcovCoverage'], shell.task([
+  'cat bin/coverage/lcov.info | ./node_modules/.bin/coveralls'
```

Comment:
Yeah, but it's only executed by travis (which is *nix).

Users that needs to build the project do not need to upload coverage manually.